### DISCUSSION TITLE
problem sinchronizzazion with node v1.1.1

### DISCUSSION THREAD

**Discussion by ivantor69** [idena-network/idena-web#412] [General]
I noticed with the latest version v1.1.1 compared to the previous one that you have solved the problem on the synchronization .. but in reality apart from this, I noticed that it takes too much time and it appears that in the log it indicates many phases where it says that it cannot connect on the indicated ip (from where it got these ip) and then the database compaction proceeds and at the end the real synchronization begins.. the time is really out of bounds especially even if I interrupt after 1 hour and the same thing always resumes and the time to complete play must pass well over 10 minutes!! I attach the log and you can see the previous one and the new one (just look at the date and [time)](url) even when it released the latest version!
[output.zip](https://github.com/idena-network/idena-web/files/12514614/output.zip)
allegated photo imagine
![immagine](https://github.com/idena-network/idena-web/assets/5076311/0e57aee0-7038-4112-a7fe-46afeb086723)
![immagine](https://github.com/idena-network/idena-web/assets/5076311/f4e9f973-0b73-4a25-a104-50522996b1e6)
![immagine](https://github.com/idena-network/idena-web/assets/5076311/e8b7d6db-fe2c-4ff4-92ce-454e4a4d407f)

I opened idena app, then closed after syncing I get 2 "clean" logs
[logs 04092023.zip](https://github.com/idena-network/idena-web/files/12514823/logs.04092023.zip)

**Comment by rioda-org** [comment]
Just use normal PC or VPS, why would anyone normal restart node every 1 hour
